src/Filters/ResultsFilter.cs

```diff
@@ -302,24 +306,21 @@ public ResultsFilter WithoutSubjects()
             };
         }
 
-
         public IEnumerable<string> GetQualificationStrings()
         {
-            // FIXME
-            if (true)
+            if (qualification.Any(x => x == QualificationOption.PgdePgceWithQts))
             {
-                yield return "QTS only";
+                yield return "Postgraduate certificate in education with qualified teacher status";
             }
-            if (true)
+            if (qualification.Any(x => x == QualificationOption.QtsOnly))
             {
-                yield return "PGCE (or PGDE) with QTS";
+                yield return "Qualified teacher status";
             }
-            if (true)
+            if (qualification.Any(x => x == QualificationOption.Other))
             {
-                yield return "Further Education (PGCE or PGDE without QTS)";
+                yield return "Other";
```

Comment:
This should be `Further Education (PGCE or PGDE without QTS)` I believ, cc @fofr can you confirm?